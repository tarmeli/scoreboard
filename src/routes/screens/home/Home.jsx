import React from "react";
import { Div } from "../../../components/Element/";

const Home = ({ data: { content } }) => <Div>{content}</Div>;

export default Home;
