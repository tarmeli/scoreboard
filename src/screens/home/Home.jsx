import React from "react";

import Div from "../../components/Element/Div/Div";

const Home = () => <Div>Hi</Div>;

export default Home;
