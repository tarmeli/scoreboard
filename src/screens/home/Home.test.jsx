import React from "react";
import { mount } from "enzyme";

import Home from "./Home";

describe("Home", () => {
    let component;

    beforeEach(() => {
        component = mount(<Home />);
    });

    it("renders", () => {
        expect(component).toMatchSnapshot();
    });
});
