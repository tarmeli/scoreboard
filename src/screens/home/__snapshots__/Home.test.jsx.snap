// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<Home>
  <Div>
    <Element
      tagName="div"
    >
      <div>
        Hi
      </div>
    </Element>
  </Div>
</Home>
`;
